homeassistant/components/kaleidescape/remote.py

```diff
@@ -0,0 +1,72 @@
+"""Sensor platform for Kaleidescape integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from kaleidescape import const as kaleidescape_const
+
+from homeassistant.components.remote import RemoteEntity
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN as KALEIDESCAPE_DOMAIN
+from .entity import KaleidescapeEntity
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+    from typing import Any
+
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import HomeAssistant
+    from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the platform from a config entry."""
+    entities = [KaleidescapeRemote(hass.data[KALEIDESCAPE_DOMAIN][entry.entry_id])]
+    async_add_entities(entities)
+
+
+VALID_COMMANDS = [
+    "select",
+    "up",
+    "down",
+    "left",
+    "right",
+    "cancel",
+    "replay",
+    "scan_forward",
+    "scan_reverse",
+    "go_movie_covers",
+    "menu_toggle",
+]
```

Comment:
```suggestion
VALID_COMMANDS = {
    "select",
    "up",
    "down",
    "left",
    "right",
    "cancel",
    "replay",
    "scan_forward",
    "scan_reverse",
    "go_movie_covers",
    "menu_toggle",
}
```

If you make this a set, `cmd in VALID_COMMANDS` can avoid a linear search

Reply:
Fixed.